### PR TITLE
Add support for new wording on War Bringer

### PR DESCRIPTION
**Pull request by madinsane**
Fixes War Bringer mod updated parsing to support new wording.

### Description of the problem being solved:
"Exerted Attacks deal 50% more Attack Damage if a Warcry Sacrificed Rage Recently" mod wording updated to be more Attack Damage instead of Damage.

### Steps taken to verify a working solution:
- Used linked build to check damage applies to attack portion but not ailments
- Checked max hit and average modes for warcry calculation correctly update
- Checked calc breakdowns correctly update

### Link to a build that showcases this PR:
```
eNrVGttu2zj2uf4KwsAsptjG1sWSnawzA8dxUgNx6rHTdvepYCTa5kQSXZFK6g7673tIyZGSWA7Nzj5sCiQSde7n8FzI9n__FkfonqScsuS0abesJiJJwEKaLE-bH28ujnrN339r9KdYrD4szjIayS-_Nd701TOKyD2JAK-JBE6XRHzaUnK_AKU1TsSKsGSC_2TpJQtPm9csIU10i5OQiu1bEGHOr3FMTpsTnOIsJGkTYR6QJByWn86AMknv5LcY02TOgjsiLlOWrRX7e0oeJiwEwPFk-mF2U2FOkypzkP1NfxrhDUnnAgvE4ddpcwAmwEtyjmP4DdRwlAEp2-nZLdfq-V3f93y32d6NPF8TEpZILbt7XAc6TclosSCBoPdkmFIxXOEkKBl6dXg7YDstby_4JIsEXUdUWuxRtFqM9y_oH9t1sDdM4Oh8Oi_perbV8n3n2Dnuws9-PCYe8aw6yLMILFrl0O11W57drRX_MxWrF0iOc9xr2T2rs18siTteJlQQA50k8pRRzpKfMMghmg5ZfEsTI0UHKcEfFnn8zXBIM14GU62zJzjBQ8ZLp_l1kFd0QZ5A1rp3NNeDm8Fm1IOUUk5JCjlD6CFIYQ9CKDjMScAgLx3C4xCUih5mzAwwR3MTBANGc1GmItepdTr5WgW0vV4d5Dn5VoJ19tCrAnZr43ecCD1yVUCrXrh7JqAU6nmu3F3-nioTnEjYcRLoEf2YpARq5n2lOPm-HkLh4NIe1l5OM7IkmppeERKsLqGCz7AgehuizFLdvcaRsFrGkYA7jFNP_inCAcaRiM-M0-rtAz7QPJ9xGmps2ISky818RUmkAa1sWUXRsmkVQTMcqigH6j26x7y6uWxvvzY5uF5wEOgjACEkzxoiq76pY3_Kli46DG2QxixLNf2RA2spMF1tOA2gsVDN7IyEWaCXiB470wm7JzFEuGpsoasuzVzfsEE_rqs4kI2igzAGQuDg7pyFS3IQk8Mx5tl6DftcRosu3gVNwcqcVkrgka8B_QFmjiFev85gyOSW0mVQQmszuKLLlUhgsNPn8gxFX5cVZvwAZUpwbRYXWfRkOqnfKOwBSK7k4MoPg57gsqGoHQsvUpJ832jTfwKuxWCUhFkqY1Sbx3OMl2z6bTXYy6dxvGapUItDHAVckRwn60ygRI3kMeXBl9tssZCDdxNYpOrUYHRxMRrejD-NCimqKPyORtGXJItv5Via_y3TypyoTIUCFkV4zQnM7AsccaBN4XEuceeQnQOhBQ8jbTHP60BDJQ1SCkbUAZZDrJbI6mRAB1JO7FqA4BUcEX2D3WzWREYA17VZkfZ0oNXErQWZj8h6CqpRXM8NJMAbLcjH9kILehSRAY1kBdSz2gQ2Ql489eChnKX0NhOa0aYGBC05ZLOspWC1IdTcHXqARVOjJUTRw-nAFhVDLyihsdB22zlZEAh3zW2v0skNviNJkSy3ibGv9hpHHDLmJYn52QYq0IU077MTmxgLCEQSX8mD0xsm8zAOBEmv8oPUgtkTMqdNkWawGJIFziK5_keGIyo2MtFXVgsSDizyFXuQnUxORqYADq65usq_DCJRUJA8tkxz_aUa6oB1oDpB9a6UVKesNAmiLIQJtKixjwJH-FbybqKlPJEdsiyBapPQSJ4m49tI2jFX4hlpSVUyftMHUQrYy4jd4sgpaZd6VQHsLUlVU8bAQZ4GJ-E8wnETfc0VHKssoSwEspFYvk-IwCEWuD0WoH5b2qCthIGnHSSUlZ9pEeQKgkVkWZMN-pY7ynGVTBOaqLPweRbHDAJgCplVYB7nsfb3qExgJwPOMN2Y67yDxlOlC6l2ap0j_wOvGf8XR5JGvpNy8v__QTUnlMc0qDWwApziNDeMhrl3ENQNsQJ1l5WLJ5WLblJCEM7tofCcYnPDS_ViRUpqSetxSECbooBJeyfQz8GD1-u5bhMJIFe517H94somR88NPQqhZofXEq8I7o-zK_XwZiXEmp-02w8PD601Fiu2IN9gVG4FLG6vgQxIeaSMeCQZtQfwc7YcqB9FqL2l1M_vfPhWc9CmVi1nj1ryVL7zzrMsp_Ou07W73jvXcTz7paLdiqL2_0jR0XDAr9liweHt9FRH4X5b-lc-XDPoIOQ3ubh9Ud7_RMkD4gRa2tVcQINNvpFUyJOu74zF_4GJy-tYrW7H9V2vB2bo5nXhPcFigtePe0QCF0XFL2oKNJLnFCyaqqK5jVUJ-G8wWcdv-ceW43Y8v7gC66sNUASjfJ6TPKgzTvKj488EUkeilit1SILCYJJSLDeBk5ffwgczWBabE_TxevzHx1FjIL7TlG4T0DnlC3wPLUjjE8YRGnwj8KDInKBpSpDbsu2WVa4NsxR0EY18riAhevzilBSKaglrVqPwyQmaHT3-aygrzcjXE9T1GtChRzSgEsZu_FUoceL8cLxfUKCaayQYmmAaI-g0oONu_NORK4oIYguUswBZihKOZP4oKdk__oIZbklOrJb3YxCGHP3qONaR43pvJZlfnS68HFtv0fYgCOV9S1WWlxQs7wioFBQ8oNDdRaFE-9V2jmz_7S8yfacEQ6uE8lMapIaexlNtAwwOR2pegBnxqd55BCBFWoU9jJMz-ZIvySNTLw8Ad1s5ayE6r0JsDwpVas7zR8S242khiixKIs_k29G0iN09cVsN0V005w94jQa3G87BnrmLkVcysraS70B9juboob3Q4gnUGWOCGwq0S5dXhDojkTgQ5QKS7x1ytURydonkmplXzwjO34TWMRPSN-Pmm1vTfsXBu-LpFQ-8JzBli0MZ7Ygk34TPK-EHHeG9PFQ7nLBrvotsE34ds1iwzSKv87p7dAx7KNld4WWouFbe8QzkMciB7mtswg3KD1UOdN9uKxvFl2eCpLUldRxhpKdnbFbHwO--2U5yzcLXPTzKOuZp3yCmbWPjuzoB0dEBMhD7J5ofTU-ae8E3iEpzN_xEuOimZImruWv0QQ0sN5PziGOSYhxj8xqmC31_mqjjatjJNndDx9hanok2OvvFPMhfEWkQZ9GO2pbPnDAyqlMcdTKiLldZsqDLF1elQcYFi2GO5ZWr1fwcB2E1aXMUEpDJs6pDeJhP6jvuXh_kDefm6V3t4NNoNrgcbW8zCkn67ef_3_u_40Fy-Q==
```

### CODE REVIEW

File: src/Modules/ModParser.lua [hunk 1]
```diff
@@ -1657,7 +1657,7 @@ local specialModList = {
 	["warcries grant (%d+) rage per (%d+) power if you have less than (%d+) rage"] = {
 		flag("Condition:CanGainRage"),
 	},
-	["exerted attacks deal (%d+)%% more damage if a warcry sacrificed rage recently"] = function(num) return { mod("ExertIncrease", "MORE", num, nil, ModFlag.Attack, 0) } end,
+	["exerted attacks deal (%d+)%% more attack damage if a warcry sacrificed rage recently"] = function(num) return { mod("ExertAttackIncrease", "MORE", num, nil, ModFlag.Attack, 0) } end,
```
Comment (Nostrademous) [inline src/Modules/ModParser.lua:1660]: Why don't we change the old modifier name `ExertIncrease` to the new one `ExertAttackIncrease` and change the mod-parse line to:
```
["exerted attacks deal (%d+)%% more ?a?t?t?a?c?k? damage if a warcry sacrificed rage recently"]
```
This prevents the need for copying all the calculations for a new-named mod that is basically the old mod and provides legacy compatibility.

File: src/Modules/CalcSections.lua [hunk 2]
```diff
@@ -371,10 +371,10 @@ return {
 	},
 	{ label = "Exerted Attacks", bgCol = colorCodes.MAINHANDBG, haveOutput = "ExertedAttackUptimeRatio",
 		{ format = "{2:output:ExertedAttackHitEffect}", { breakdown = "ExertedAttackHitEffect"}, },
-		{ format = "{2:output:ExertedAttackAvgDmg}", { modName = "ExertIncrease", cfg = "skill" }, },
+		{ format = "{2:output:ExertedAttackAvgDmg}", { modName = "ExertIncrease", cfg = "skill" }, { modName = "ExertAttackIncrease", cfg = "skill" }, },
```
Comment (Nostrademous) [inline src/Modules/CalcSections.lua:374]: not needed if last comment is done

File: src/Modules/CalcSections.lua [hunk 3]
```diff
@@ -371,10 +371,10 @@ return {
...
 		{ format = "{0:output:ExertedAttackUptimeRatio}%", { breakdown = "ExertedAttackUptimeRatio" }, },
 		{ format = "" },
-		{ format = "{2:output:ExertedAttackMaxHitEffect}", { modName = "ExertIncrease", cfg = "skill" }, },
+		{ format = "{2:output:ExertedAttackMaxHitEffect}", { modName = "ExertIncrease", cfg = "skill" }, { modName = "ExertAttackIncrease", cfg = "skill" }, },
```
Comment (Nostrademous) [inline src/Modules/CalcSections.lua:377]: not needed if last comment is done

File: src/Modules/CalcOffence.lua [hunk 4]
```diff
@@ -1038,6 +1038,10 @@ function calcs.offence(env, actor, activeSkill)
 				local mod = value.mod
 				skillModList:NewMod("Damage", mod.type, mod.value, mod.source, mod.flags, mod.keywordFlags)
 			end
+			for _, value in ipairs(env.modDB:Tabulate("MORE", skillCfg, "ExertAttackIncrease")) do
+				local mod = value.mod
+				skillModList:NewMod("Damage", mod.type, mod.value, mod.source, mod.flags, mod.keywordFlags)
+			end
```
Comment (Nostrademous) [inline src/Modules/CalcOffence.lua:1044]: entire `for` loop not needed if last comment is done

File: src/Modules/CalcOffence.lua [hunk 5]
```diff
@@ -1854,14 +1858,18 @@ function calcs.offence(env, actor, activeSkill)
 				if globalOutput.ExertedAttackUptimeRatio > 0 then
 					local incExertedAttacks = skillModList:Sum("INC", cfg, "ExertIncrease")
 					local moreExertedAttacks = skillModList:Sum("MORE", cfg, "ExertIncrease")
+					local moreExertedAttackDamage = skillModList:Sum("MORE", cfg, "ExertAttackIncrease")
```
Comment (Nostrademous) [inline src/Modules/CalcOffence.lua:1861]: not needed if last comment is done

File: src/Modules/CalcOffence.lua [hunk 6]
```diff
@@ -1854,14 +1858,18 @@ function calcs.offence(env, actor, activeSkill)
...
 					if activeSkill.skillModList:Flag(nil, "Condition:WarcryMaxHit") then
 						skillModList:NewMod("Damage", "INC", incExertedAttacks, "Exerted Attacks")
 						skillModList:NewMod("Damage", "MORE", moreExertedAttacks, "Exerted Attacks")
+						skillModList:NewMod("Damage", "MORE", moreExertedAttackDamage, "Exerted Attack Damage", ModFlag.Attack)
```
Comment (Nostrademous) [inline src/Modules/CalcOffence.lua:1865]: not needed if last comment is done

File: src/Modules/CalcOffence.lua [hunk 7]
```diff
@@ -1854,14 +1858,18 @@ function calcs.offence(env, actor, activeSkill)
...
 					else
 						skillModList:NewMod("Damage", "INC", incExertedAttacks * globalOutput.ExertedAttackUptimeRatio / 100, "Uptime Scaled Exerted Attacks")
 						skillModList:NewMod("Damage", "MORE", moreExertedAttacks * globalOutput.ExertedAttackUptimeRatio / 100, "Uptime Scaled Exerted Attacks")
+						skillModList:NewMod("Damage", "MORE", moreExertedAttackDamage * globalOutput.ExertedAttackUptimeRatio / 100, "Uptime Scaled Exerted Attack Damage", ModFlag.Attack)
```
Comment (Nostrademous) [inline src/Modules/CalcOffence.lua:1869]: not needed if last comment is done

File: src/Modules/CalcOffence.lua [hunk 8]
```diff
@@ -1854,14 +1858,18 @@ function calcs.offence(env, actor, activeSkill)
...
 					end
 					globalOutput.ExertedAttackAvgDmg = calcLib.mod(skillModList, skillCfg, "ExertIncrease")
+					globalOutput.ExertedAttackAvgDmg = globalOutput.ExertedAttackAvgDmg * calcLib.mod(skillModList, skillCfg, "ExertAttackIncrease")
```
Comment (Nostrademous) [inline src/Modules/CalcOffence.lua:1872]: not needed if last comment is done

**Review by Nostrademous — CHANGES_REQUESTED** [review]

**Comment by Lilylicious** [comment]
Fixes #4014

**Review by Nostrademous — APPROVED** [review]

**Review by Wires77 — APPROVED** [review]
:+1:

**Pull request merged**